### PR TITLE
Create sample configuration file and README

### PR DESCRIPTION
**Pull request by rbelem**
Create sample configuration files and README based on mx-puppet-slack.

### CODE REVIEW

**Comment by Sorunome** [comment]
Eeeeer, the in #1 other docker stuffs was PR'd and it MCs now, uuuuh, would you mind fixing the MC? Thank you ^-^

File: README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+*   Restart synapse.
+*   Start the bridge:
+    ```
+    npm run start
+    ```
+*   Start a direct chat with the bot user (`@_skypepuppet_bot:domain.tld` unless you changed the config).
+    (Give it some time after the invite, it'll join after a minute maybe.)
+*   Get your Skype username and password as below, and tell the bot user to link your skype account:
+    ```
+    link <username> <password> (see below for details)
```
Comment (Sorunome) [inline README.md:46]: Random `(see below for details)`, what's up with this?

File: sample.config.yaml [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+# Map certain homeserver URLs to the C-S API endpoint
+# Useful for double-puppeting if .well-known is unavailable for some reason
+#homeserverUrlMap:
+#  yourserver.com: http://localhost:1234
+
+namePatterns:
+  # Override the protocols set default name patterns
+  # Which variables are available depends on protocol implementation
+  user: :name
+  userOverride: :name
```
Comment (Sorunome) [inline sample.config.yaml:122]: `userOverride` feature also isn't used in skype :)

File: sample.config.yaml [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+  interval: 500
+  # if the im.vector.user_status state setting should be diabled
+  #disableStatusState: false
+  # A blacklist of remote user IDs for the im.vector.user_status state setting
+  #statusStateBlacklist:
+  # - baduser
+
+relay:
+  # Enable/Disable relay bridging mode
+  enabled: false
```
Comment (Sorunome) [inline sample.config.yaml:105]: uuuh, the `enabled` flag is an old key. Might be that soru forgot to remove it from the bridge library.

File: README.md (hunk 1 above)
Comment (rbelem) [reply README.md]: Oops! Missed that.

File: sample.config.yaml (hunk 2 above)
Comment (rbelem) [reply sample.config.yaml]: Removed!

File: sample.config.yaml (hunk 3 above)
Comment (rbelem) [reply sample.config.yaml]: Removed!

**Comment by Sorunome** [comment]
Thanks a log for bearing through this and for the contribution ^.^

**Pull request merged**